src/lib/y2storage/storage_feature.rb

```diff
@@ -0,0 +1,226 @@
+# Copyright (c) [2016-2017,2019-2020] SUSE LLC
```

Comment:
Are these years OK?

Reply:
Well, the original `StorageFeatures` class was basically copied from the old storage. And some people tries to always add the exact year when they touch a file... but TBH I also find it weird. I will change it.

Reply:
Done in a later commit already added to this PR.